src/account/components/AccountProfile.test.js

```diff
@@ -1,11 +1,11 @@
-import React from 'react';
+import AccountProfile from 'account/components/AccountProfile';
```

Comment:
thought: This doesn't seem better to me, this is mixing Visual Components, with non visual dependencies, and owned dependencies with thrid party dependencies.

question: What are the advantages you see on this approach?

Reply:
The idea is to be consistent across files and, within a file, across functions within an import. We already do this in our Python code.

If the default order isn't good, we can customize it.